libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -140,6 +140,7 @@ internal class MapboxTripSession(
     private val fallbackVersionsObservers = CopyOnWriteArraySet<FallbackVersionsObserver>()
 
     private val bannerInstructionEvent = BannerInstructionEvent()
+    private var lastVoiceInstruction: VoiceInstructions? = null
```

Comment:
I guess it should be set back to null in some cases. When don't we want to fallback to the previous value? I guess after route update. @LukasPaczos , do you know any scenarios when we don't want to fallback to the previous voice instruction?

Reply:
That's correct, similarly to the `bannerInstrucitons`.